### ISSUE TITLE
econnreset error

### ISSUE THREAD

**Issue by realjax** [eschava/node-red-contrib-fcm-receive-node#1]
Getting this every few seconds.

```
Error: read ECONNRESET
    at TLSWrap.onStreamRead (internal/stream_base_commons.js:205:27) {
  errno: 'ECONNRESET',
  code: 'ECONNRESET',
  syscall: 'read'
}
```

**Comment by eschava** [comment]
yes, I know about it and filed an issue to the parent push-receiver library
https://github.com/MatthieuLemoine/push-receiver/issues/33

**Comment by realjax** [comment]
Hm, I checked his code and it seems this happens in client.js  at  line 145 in this function
```
_onSocketError(error) {
    // ignore, the close handler takes care of retry
    console.error(error);
  }
```
It says ignore and then it doesn't ignore it :)

Although the error is handled, it is still being sent to the console ( dunno why to be honest) and then that bubbles up to the log of Node-red ( which is where console.log() messages of any npm module will show )

So, not very clean but also not a problem.  I have removed the line from client.js in my node_modules folder, which stops the cluttering of the logging, but the author of push-receiver should fix it permanently.

**Comment by eschava** [comment]
Thanks for your investigation! You are my hero :)

**Comment by realjax** [comment]
No problem. I also created a pull request on his repo but I doubt he'll merge it. Looks like that project is abandoned.

**Comment by eschava** [comment]
Should be fixed with the new release of push-receiver library
